### ISSUE TITLE
Request for fix: MissingFieldInEquals false positive for equals without parameters

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#63]
The next has a false positive, it should be ignored since a non-standard equals is used. (Standard equals has 1 argument)

```
class EqTryoutOtherEquals {
    String field1o;
    String field2o;

    public EqTryout2(String arg2) {
        field2 = arg2;
    }
    public void setField1(String arg1) {
        field1 = arg1;
    }
    public boolean equals() {
        field1o = "";
        return false;
    }
}
```

**Comment by jborgers** [comment]
Mentioned #62 in commit where it should have been this #63

**Comment by jborgers** [comment]
Fixed
